common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -348,3 +356,143 @@ def test_user_is_under_age(self):
         Then i should see message `You must be over 13 to share a full profile.`
         """
         self.verify_profile_forced_private_message('2010', message='You must be over 13 to share a full profile.')
+
+    def test_user_can_only_see_default_image_for_private_profile(self):
+        """
+        Scenario: Default profile image behaves correctly for under age user.
+
+        Given that I am on my profile page with private access
+        And I can see default image
+        When I move my cursor to the image
+        Then i cannot see the upload/remove image text
+        And i cannot upload/remove the image.
+        """
+        self.visit_my_profile_page(self.MY_USER, privacy=self.PRIVACY_PRIVATE)
+
+        self.verify_profile_forced_private_message('2010', message='You must be over 13 to share a full profile.')
```

Comment:
Don't hard code 2010 as a birth year that is always underage as that will stop working in the future. Compute it from the current year instead.